resumebuilder/src/components/Education.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Education = (props) => {
+const { applicant } = props
```

Comment:
You can do destructuring directly in your function params to avoid more line

```suggestion
const Education = ({ applicant }) => {
```

Reply:
Yeah I just saw that in someone elses code today! Will definitely do this in the future! 🙏